src/msg/simple/Accepter.cc

```diff
@@ -304,6 +307,14 @@ void *Accepter::entry()
   pfd[0].events = POLLIN | POLLERR | POLLNVAL | POLLHUP;
   pfd[1].fd = shutdown_rd_fd;
   pfd[1].events = POLLIN | POLLERR | POLLNVAL | POLLHUP;
+  
+  std::stringstream ss;
+  ss << "Accepter thread " << (void *)pthread_self();
```

Comment:
It's pretty trivial, but since we only use this when generating an ```hb```, let's generate it in the below if-block. (That also means the if-block gets braces — we have plenty of naked one-liners without them, but I prefer the braces for clarity when possible.)

Reply:
Yes, can not agree more!